pkg/statistics/handle/types/interfaces.go

```diff
@@ -375,8 +376,9 @@ type NeededItemTask struct {
 type StatsLoad struct {
 	NeededItemsCh  chan *NeededItemTask
 	TimeoutItemsCh chan *NeededItemTask
-	WorkingColMap  map[model.TableItemID][]chan stmtctx.StatsLoadResult
-	SubCtxs        []sessionctx.Context
+
+	Singleflight singleflight.Group
```

Comment:
Could you please add some comment for this param?

Reply:
I have updated the comment.